src/Sylius/Bundle/ResourceBundle/Controller/ResourcesResolver.php

```diff
@@ -46,4 +60,42 @@ public function getResources(RequestConfiguration $requestConfiguration, Reposit
 
         return $repository->findBy($criteria, $sorting, $requestConfiguration->getLimit());
     }
+
+    /**
+     * @param array $sorting
+     * @param RepositoryInterface $repository
+     *
+     * @return array
+     */
+    private function unsetUnexistSortingFields(array $sorting, RepositoryInterface $repository)
```

Comment:
The only difference between these methods is first argument name... We can merge them